packages/playground/Samples/text.tsx

```diff
@@ -5,13 +5,17 @@
  */
 
 import React from 'react';
-import {AppRegistry, StyleSheet, Text, View} from 'react-native';
+import {AppRegistry, StyleSheet, View} from 'react-native';
+import {Text} from 'react-native-windows';
```

Comment:
We'll want to figure out how to get this to work without having to import react-native-windows before we merge this PR.

Reply:
Did we figure this out? If not, we will want to convert this back to a draft PR!

It looks like we override ViewAccessibility.d.ts too, can you see if that's being pointed to correctly?

Reply:
You'll have to import from react-native-windows to get the windows specific types.  Thats the same as all our windows specific types.